extensions/default/src/customizations/studyBrowserCustomization.ts

```diff
@@ -88,6 +88,36 @@ export default {
             });
           }
 
+          commandsManager.run('setDisplaySetsForViewports', {
+            viewportsToUpdate: updatedViewports,
+          });
+        },
+    ],
+  },
+  'studyBrowser.thumbnailClickCallback': {
+    callbacks: [
+      ({ activeViewportId, servicesManager, commandsManager, isHangingProtocolLayout }) =>
```

Comment:
This action should be added to a commands module, and the customization should simply indicate which command should be called.  That way the code doesn't get repeated, and it is as easy as just specifying the command name to use.
If you do that for double click and single click, then either one can be bound to load the display set without duplicating how it is handled, and you can also do custom behaviour like loading the DICOM SR on double click, and displaying it with single click.